sam/template.yaml

```diff
@@ -0,0 +1,44 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: 'AWS::Serverless-2016-10-31'
+Description: MXNet Lambda Function 
+Resources:
+  ApiGatewayApi:
+    Type: AWS::Serverless::Api
+    Properties:
+      DefinitionUri: ./swagger.yaml #UPDATE
+      StageName: prod
+      Variables:
+        LambdaFunctionName: !Ref LambdaFunction
+
+  LambdaFunction:
+    Type: 'AWS::Serverless::Function'
+    Properties:
+      Handler: lambda_function.lambda_handler
+      Runtime: python2.7
+      CodeUri: ./lambda_function.zip #UPDATE
```

Comment:
How about storing the zip file on S3 & adding that here instead?
So, parameters would be:
`Code:
  S3Bucket:
 S3Key:
`
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-lambda-function-code.html

Reply:
SAM actually do that for you with `aws cloudformation package` which automatically upload your `lambda_function.zip` to S3 bucket and generate a `template-out.yaml` which is the actual cloudformation that is used to define the stack. 

Good idea though!